x/tokenfactory/keeper/before_send_test.go

```diff
@@ -328,20 +362,44 @@ func (suite *KeeperTestSuite) TestInfiniteTrackBeforeSend() {
 			_, err = suite.msgServer.SetBeforeSendHook(suite.ChainA.GetContext(), types.NewMsgSetBeforeSendHook(suite.ChainA.SenderAccounts[0].SenderAccount.GetAddress().String(), factoryDenom, cosmwasmAddress.String()))
 			suite.Require().NoError(err, "test: %v", tc.name)
 
-			ctx := suite.ChainA.GetContext().WithGasMeter(storetypes.NewGasMeter(30_000_000))
+			ctx := suite.ChainA.GetContext().WithGasMeter(storetypes.NewGasMeter(tc.gasLimit))
 
 			if tc.blockBeforeSend {
```

Comment:
Why, in the case where tc.blockBeforeSend == true, are we testing transfers between regular accounts, while in the case where tc.blockBeforeSend == false, we’re testing transfers between module accounts?